src/js/pages/services/DeploymentsTab.js

```diff
@@ -1,73 +1,158 @@
+import ServicesBreadcrumb from '../../components/ServicesBreadcrumb';
+import StringUtil from '../../utils/StringUtil';
 
 const columnClassNameGetter = ResourceTableUtil.getClassName;
 const columnHeading = ResourceTableUtil.renderHeading({
-  id: 'DEPLOYMENT',
+  id: 'AFFECTED SERVICES',
   startTime: 'STARTED',
+  location: 'LOCATION',
   status: 'STATUS'
 });
-const columns = [
-  {
-    className: columnClassNameGetter,
-    heading: columnHeading,
-    prop: 'id',
-    render: function (prop, deployment) {
-      return deployment.getId();
-    }
-  },
-  {
-    className: columnClassNameGetter,
-    heading: columnHeading,
-    prop: 'startTime',
-    render: function (prop, deployment) {
-      return moment(deployment.getStartTime()).fromNow();
-    }
-  },
-  {
-    className: columnClassNameGetter,
-    heading: columnHeading,
-    prop: 'status',
-    render: function (prop, deployment) {
-      return `Step ${deployment.getCurrentStep()} of ${deployment.getTotalSteps()}`;
-    }
-  }
+const METHODS_TO_BIND = [
+  'renderAffectedServices',
+  'renderAffectedServicesList',
+  'renderLocation',
+  'renderStartTime',
+  'renderStatus'
 ];
 
-class DeploymentsTab extends React.Component {
+class DeploymentsTab extends mixin(StoreMixin) {
 
   constructor() {
     super(...arguments);
-    this.onDeploymentsChange = this.onDeploymentsChange.bind(this);
 
-    this.state = {
-      deployments: MarathonStore.get('deployments')
-    };
+    this.state = {deployments: DCOSStore.deploymentsList};
+    this.store_listeners = [
+      {name: 'dcos', events: ['change'], suppressUpdate: true}
+    ];
+
+    METHODS_TO_BIND.forEach((method) => {
+      this[method] = this[method].bind(this);
+    }, this);
+  }
+
+  onDcosStoreChange() {
+    this.setState({deployments: DCOSStore.deploymentsList});
+  }
+
+  renderAffectedServices(prop, deployment) {
+    return (
+      <dl className="deployment-services-list flush-top flush-bottom tree-list">
+        <dt className="deployment-id text-uppercase">{deployment.id}</dt>
+        {this.renderAffectedServicesList(deployment.getAffectedServices())}
+      </dl>
+    );
+  }
+
+  renderAffectedServicesList(services) {
+    return services.map(function (service, index) {
+      const id = encodeURIComponent(service.getId());
+      const image = service.getImages()['icon-small'];
+
+      return (
+        <dd key={index}>
+          <span className="icon icon-small icon-image-container icon-app-container deployment-service-icon">
+            <img src={image} />
+          </span>
+          <Link to="services-detail" params={{id}}>
+            {StringUtil.capitalize(service.getName())}
+          </Link>
+        </dd>
+      );
+    });
+  }
+
+  renderLocation(prop, deployment) {
+    const services = deployment.getAffectedServices();
+    const items = services.map(function (service, index) {
+      return (
+        <li key={index}>
+          <ServicesBreadcrumb
```

Comment:
We should be using the new `Breadcrumbs` component, all of the new components will be using it and we will be removing the old `ServicesBreadcrumb`. See this #416